src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,27 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+
+interface SingleMovie {
```

Comment:
In our project we already had file `react-app-env.d.ts` for putting there interfaces. Please move it.